### DISCUSSION TITLE
Vite problem with images

### DISCUSSION THREAD

**Discussion by thomasnik** [vitejs/vite-plugin-react#230] [Q&A] [answered]
Hi, Im running ionic react application using vite. When i try to render cesium with resium library, vite says that cannot find the images and some js files from the components of resium library. The images are into the node_modules/cesium/Widgets/Images/ImageryProviders/ but vite is looking to the node_modules/.vite/deps/Widgets/Images/ImageryProviders/

Please does anyone know what i have to do??? It seems that the images are not included into the vite bundle of files

**Comment by thomasnik** [comment, marked as answer]
https://www.npmjs.com/package/vite-plugin-cesium the solution

**Reply by ArnaudBarre** [reply]
This plugin seems good. I think it would be better to redirect to imports via resolveId than middleware + serveStatic, but this not a big issue
